test/test_slave.html

```diff
@@ -19,6 +19,7 @@
     <title>PDF.js test slave</title>
     <meta charset="utf-8">
     <script src="../build/generic/build/pdf.js"></script>
+    <script src="../coverage/lcov-report/pdf.js/build/generic/build/pdf.js"></script>
```

Comment:
The query parameter is available in `location.search`. You can replace line 21 and 22 with the following to conditionally load a different version of the library depending on the query string parameter (change the paths if necessary):
```html
<script>
if (location.search.replace('?', '&').includes('coverage=true')) {
    document.write('<script src="../coverage/build/generic/build/pdf.js"></script>');
} else {
    document.write('<script src="../build/generic/build/pdf.js"></script>');
}
</script>
```